kolibri/plugins/learn/assets/src/views/SearchBox.vue

```diff
@@ -291,6 +299,17 @@
         message: 'Apps',
         context: 'Learners can filter their searches for resources by type. In this case, apps.',
       },
+      /* eslint-disable kolibri/vue-no-unused-translations */
+
+      searchPlaceholder: {
```

Comment:
You have the core string 🙂 

https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/commonCoreStrings.js#L308